src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6056,7 +6056,8 @@ void handleInitializer(IOperation innerInitializer)
 #if DEBUG
                         var validKinds = ImmutableArray.Create(OperationKind.Invocation, OperationKind.DynamicInvocation, OperationKind.Increment, OperationKind.Literal,
                                                                OperationKind.LocalReference, OperationKind.Binary, OperationKind.FieldReference, OperationKind.Invalid,
-                                                               OperationKind.InterpolatedString);
+                                                               OperationKind.InterpolatedString, OperationKind.PropertyReference, OperationKind.FieldReference,
```

Comment:
>OperationKind.FieldReference [](http://example.com/codeflow?start=130&length=28)

It looks like this value is already added above.

Reply:
Removed those added values